src/client/src/api/fetchApiV2.js

```diff
@@ -413,6 +413,10 @@ export const getBoreholeGeometryDepthTVD = async (boreholeId, depthMD) => {
   return await fetchApiV2(`boreholegeometry/getDepthTVD?boreholeId=${boreholeId}&depthMD=${depthMD}`, "GET");
 };
 
+export const getBoreholeGeometryDepthMasl = async (boreholeId, depthMD) => {
+  return await fetchApiV2(`boreholegeometry/GetDepthInMasl?boreholeId=${boreholeId}&depthMD=${depthMD}`, "GET");
```

Comment:
PP: sonst im file ist auch alles lower case
```suggestion
  return await fetchApiV2(`boreholegeometry/getdepthinmasl?boreholeId=${boreholeId}&depthMD=${depthMD}`, "GET");
```